MykSuite/src/main/java/com/infomaniak/lib/myksuite/ui/screens/MyKSuiteDashboardScreen.kt

```diff
@@ -0,0 +1,238 @@
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.infomaniak.lib.myksuite.R
+import com.infomaniak.lib.myksuite.ui.components.MyKSuiteChip
+import com.infomaniak.lib.myksuite.ui.components.WeightOneSpacer
+import com.infomaniak.lib.myksuite.ui.screens.components.AppStorageQuotas
+import com.infomaniak.lib.myksuite.ui.screens.components.ExpendableActionItem
+import com.infomaniak.lib.myksuite.ui.screens.components.LimitedFunctionalities
+import com.infomaniak.lib.myksuite.ui.screens.components.UserAvatar
+import com.infomaniak.lib.myksuite.ui.theme.Dimens
+import com.infomaniak.lib.myksuite.ui.theme.Margin
+import com.infomaniak.lib.myksuite.ui.theme.MyKSuiteTheme
+
+@Composable
+fun MyKSuiteDashboardScreen(
+    userName: String,
+    avatarUri: String = "",
+    dailySendingLimit: () -> String,
+    onClose: () -> Unit = {},
+) {
+    Box(
+        Modifier
+            .verticalScroll(rememberScrollState())
+            .background(MyKSuiteTheme.colors.background),
+    ) {
+        Image(
+            modifier = Modifier
+                .fillMaxWidth()
+                .fillMaxHeight(0.45f),
+            contentScale = ContentScale.FillBounds,
+            imageVector = ImageVector.vectorResource(R.drawable.illu_dashboard_background),
+            contentDescription = null,
+        )
+        Column(Modifier.fillMaxSize()) {
+
+            val paddedModifier = Modifier.padding(horizontal = Margin.Medium)
+
+            TopAppBar(onClose)
+            SubscriptionInfoCard(paddedModifier, avatarUri, userName, dailySendingLimit)
+            // TODO: Add this line when we'll have In-app payments
+            //  MyKSuitePlusPromotionCard(paddedModifier) {}
+        }
+    }
+}
+
+@Composable
+private fun TopAppBar(onClose: () -> Unit) {
+    Row(
+        modifier = Modifier.padding(bottom = Margin.Medium, start = Margin.Mini, end = Margin.Mini),
+        verticalAlignment = Alignment.CenterVertically,
+    ) {
+        IconButton(
+            modifier = Modifier.size(Dimens.iconButtonSize),
+            onClick = onClose,
+            colors = IconButtonDefaults.iconButtonColors(contentColor = MyKSuiteTheme.colors.iconColor),
+        ) {
+            Icon(
+                modifier = Modifier.size(Dimens.iconSize),
+                imageVector = ImageVector.vectorResource(R.drawable.ic_cross_thick),
+                contentDescription = stringResource(R.string.buttonClose),
+            )
+        }
+        Text(
+            modifier = Modifier
+                .padding(end = Dimens.iconButtonSize)
+                .fillMaxWidth(),
+            text = stringResource(R.string.myKSuiteDashboardTitle),
+            style = MyKSuiteTheme.typography.h2,
+            color = MyKSuiteTheme.colors.primaryTextColor,
+            textAlign = TextAlign.Center,
+        )
+    }
+}
```

Comment:
You need to use the material component called TopAppBar instead of recreating it all over from zero